packages/babel-generator/test/index.js

```diff
@@ -348,6 +348,20 @@ describe("generation", function () {
     const output = generate(type).code;
     expect(output).toBe("(infer T)[]");
   });
+
+  it("comments with same start index", () => {
+    const code1 = "/* leading a */ a();";
```

Comment:
Is there a particular reason we can't construct and parse the full program in the test instead of cobbling it together after?